### DISCUSSION TITLE
Reliability Indexes Calculation

### DISCUSSION THREAD

**Discussion by pedrotorres08** [e2nIEE/pandapower#1642] [Ideas]
I wonder if it is planned to include reliability calculation in pandapower (e.g. to calculate expected SAIDI and SAIFI given equipment and lines failure rates, etc)

**Comment by lthurner** [comment]
Such algorithms have been implemented at Fraunhofer IEE / University of Kassel, but are not part of the open source pandapower package. I am not sure if there are plans to include this in the open source package or not. @rbolgaryn ?
